test/lib/pineInit.ts

```diff
@@ -0,0 +1,48 @@
+import * as express from 'express';
+import { exit } from 'process';
+import * as pine from '../../src/server-glue/module';
+import { Client } from 'pg';
+
+export async function init(
+	initConfig: any,
+	initPort: number,
+	deleteDb: boolean = false,
+) {
+	const app = express();
+	app.use(express.urlencoded({ extended: true }));
+	app.use(express.json());
+
+	app.use('/ping', (_req, res) => {
+		res.sendStatus(200);
+	});
+
+	try {
+		await cleanInit(deleteDb);
+		await pine.init(app, initConfig);
+		await new Promise((resolve) => {
+			app.listen(initPort, () => {
+				resolve('server started');
+			});
+		});
+		return app;
+	} catch (e) {
+		console.log(`pineInit ${e}`);
+		exit(1);
+	}
+}
+
+async function cleanInit(deleteDb: boolean = false) {
+	if (!deleteDb) {
+		return;
+	}
+
+	try {
+		const client = new Client(process.env.DATABASE_URL);
```

Comment:
Any reason you're calling the client directly rather than using the database-layer module from within pinejs?

Reply:
No other than that I've not checked it. Big thanks!